go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/emicklei/go-restful v2.9.5+incompatible
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
+	github.com/go-openapi/spec v0.20.6
```

Comment:
is this dep ok @liggitt

Reply:
definitely not (and it's not actually required... the CI verification job is showing it as removed when vendor is regenerated)

make sure you have up-to-date generated files (`make generated`) to make sure you don't have old stale references to this dep in generated openapi go files, then run `hack/update-vendor.sh` again

Reply:
thanks 
Run  this again:
```
make generated_files 
hack/pin-dependency.sh sigs.k8s.io/structured-merge-diff/v4 v4.2.3
hack/update-internal-modules.sh
hack/update-vendor.sh
```